spec/features/ability_spec.rb

```diff
@@ -226,77 +229,5 @@
 
     visit admin_conference_commercials_path(conference3.short_title)
     expect(current_path).to eq(admin_conference_commercials_path(conference3.short_title))
-
-  end
-
-  scenario 'when user is volunteer coordinator' do
```

Comment:
Because we don't have an ability based on volunteer coordinator. Just for dead objects (Vposition, Vday) which we don't expose anymore